observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -16,6 +16,63 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+The following table provides information about which features are available depending on the version of {coo-full} and {product-title}:
+
+[cols="^1,^1,^1,^1,^1,^1", options="header"]
+|===
+| *COO Version*   | *OCP Versions*       | *Dashboards*   | *Distributed Tracing*   | *Logging*   | *Troubleshooting Panel* 
```

Comment:
Nitpick: I don't think you need to worry about adding the bold markup to table headers because from my experience with tables, table headers are bold by default.

Reply:
+1 should leave them as is for consistency across all our tables. I also don't think I'd recommend centering them, again for consistency across all our tables

Reply:
done